mathics/builtin/specialfns/bessel.py

```diff
@@ -691,7 +701,18 @@ class SphericalBesselY(_Bessel):
 
     attributes = ("Listable", "NumericFunction", "Protected")
 
-    rules = {"SphericalBesselY[n_, z_]": "Sqrt[Pi / 2] / Sqrt[z] BesselY[n + 0.5, z]"}
+    rules = {
+        "SphericalBesselY[n_, z_]": """
+             Times[
+                Rational[1, 2], 
+                Power[Pi, Rational[1, 2]], 
+                Power[2, Rational[1, 2]], 
+                Power[z, Rational[-1, 2]], 
+                BesselY[Plus[Rational[1, 2], n], z]
+               ]
+             """,
+        "N[SphericalBesselY[n_,z_]]": "Sqrt[Pi/2]/Sqrt[z] * BesselY[N[1/2+n], N[z]]",
```

Comment:
I'd suggest adding a comment here and in the other place where an additional rule is added to suggest why this is beneficial.